README.md

```diff
@@ -59,6 +59,7 @@ User documentation:
 * [Installation guide](doc/installation.md) - Describes the installation process for different systems and .NET versions
 * Configuration:
   * [Main XML configuration file](doc/xmlConfigFile.md)
+  * [Main YAML configuration file](doc/YamlConfigFile.md)
   * [EXE configuration file](doc/exeConfigFile.md)
```

Comment:
Could you remove the "EXE configuration file" by the way? We no longer use it.

Reply:
np problem